src/pages/reference/extending-unified-schema.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Extending the Unified Schema
+description: Learn how to extend the unified schema with resolvers.
+---
+Passive: Bob Dylan was injured in a motorcycle accident.
```

Comment:
copy/paste error in lines 5- 9?

Reply:
ah! was testing to see if i could find a passive sentence plugin that could actually recognize a passive sentence. no luck, though.